src/components/TaskCard/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import PropTypes from "prop-types";
+import style from "components/TaskCard/index.module.scss";
+const TaskCard = ({ task, createdTime }) => {
+  return (
+    <div className={style.container}>
+      <p>{task}</p>
+      <p>{`Created: ${createdTime}`}</p>
+    </div>
+  );
+};
+TaskCard.propTypes = {
+  task: PropTypes.string.isRequired,
+  createdTime: PropTypes.string.isRequired,
```

Comment:
Can we validate `createdTime` string contains a date rather validating as string ?